bundles/admin/admin-layereditor/view/AdminLayerFormService.js

```diff
@@ -372,6 +373,34 @@ export class AdminLayerFormService {
         });
     }
 
+    fetchRolesAndPermissionTypes () {
+        const me = this;
+        this.loading = true;
+        fetch(Oskari.urls.getRoute('GetAllRolesAndPermissionTypes')).then(response => {
+            if (response.ok) {
+                return response.json();
+            } else {
+                return Promise.reject(Error('Fetching user roles and permission types failed'));
```

Comment:
Should this be "new Error" instead of just "Error"?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error states "When Error is used like a function -- without new, it will return an Error object. Therefore, a mere call to Error will produce the same output that constructing an Error object via the new keyword would." I admit that it is clearer for most readers if new key word is included.

Reply:
new keyword added